.changeset/minor-coffee-enegery.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-studio': patch
+---
+
+Leverage `externalFormats` apis to dynamically fetch available `externalFormat` types.
+`WIP` Add model generation flow to `SchemaSet` editor to allow schema imports.
```

Comment:
if it's substantial, we can keep this and remove `WIP`, if not, let's add changeset next time we complete this, I don't think we want `WIP` in changelog